Formula/noti.rb

```diff
@@ -17,19 +16,20 @@ class Noti < Formula
   depends_on "go" => :build
 
   def install
-    mkdir_p "#{buildpath}/src/github.com/variadico"
-    ln_s buildpath, "#{buildpath}/src/github.com/variadico/noti"
     ENV["GOPATH"] = buildpath
-    cd "#{buildpath}/src/github.com/variadico/noti" do
-      if build.stable?
-        system "go", "build", "-o", "#{bin}/noti"
-      else
-        system "go", "build", "-o", "#{bin}/noti", "cmd/noti/main.go", "cmd/noti/osx.go"
-      end
+
+    # Move files to right location in GOPATH.
```

Comment:
@DomT4 Do you mean this comment? I added this because I don't touch Ruby/Homebrew very often so I had to look up what `buildpath` and `install Dir` do. To me, it's not very obvious what those lines do.